tools/tcoffee/tcoffee.xml

```diff
@@ -0,0 +1,186 @@
+<tool id="tcoffee" name="T-Coffee" version="0.1">
+    <requirements>
+        <requirement type="package" version="11.0.8">t_coffee</requirement>
+    </requirements>
+    <command><![CDATA[
+        t_coffee
+
+        -in "$input"
+
+        #if "$method1" != "":
+            M"$method1"
+        #end if
+
+        #if "$method2" != "":
+            M"$method2"
+        #end if
+
+        #if "$method3" != "":
+            M"$method3"
+        #end if
+
+        #if $out:
+            -output
+            $out.format_msf_aln
+            $out.format_clustalw_aln
+            $out.format_pir_aln
+            $out.format_fasta_aln
+            $out.format_phylip
+            $out.format_pir_seq
+            $out.format_fasta_seq
+            $out.format_score_ascii
+            $out.format_score_html
+        #end if
+]]>
+    </command>
+    <inputs>
+        <param name="input" type="data" format="fasta" label="Source file" />
+
+        <param name="method1" type="select" label="Pairwise Sequence Alignment Method">
```

Comment:
Do you know what default is? You can use a `type="select" optional="True"` and remove the Default option.